Sources/NIOSSH/TransportProtection/AESGCM.swift

```diff
@@ -342,3 +342,18 @@ extension Data {
         self = self[contentStartIndex ..< contentEndIndex]
     }
 }
+
+extension AES.GCM.SealedBox {
+    fileprivate init(nonce: SSHAESGCMNonce, ciphertext: ByteBufferView, tag: ByteBufferView) throws {
+        // As a workaround for a Swift Crypto inefficiency, we create the combined representation
+        // directly.
+        var combined: [UInt8] = []
+        combined.reserveCapacity(12 + ciphertext.count + tag.count)
```

Comment:
can we add a comment that `12` is the size of `nonce` or can we make it a static property on `SSHAESGCMNonce`?

Reply:
Done.